Non-RPC/mumble-ping.py

```diff
@@ -2,14 +2,20 @@
 # Based on pcgod's mumble-ping script found at http://0xy.org/mumble-ping.py.
 
 from struct import *
+from string import Template
 import socket, sys, time, datetime
 
 if len(sys.argv) < 3:
-    print(f"Usage: {sys.argv[0]} <host> <port>")
+    print(f"Usage: {sys.argv[0]} <host> <port> [<format>]")
     sys.exit()
 
 host = sys.argv[1]
 port = int(sys.argv[2])
+try:
+    fmt = sys.argv[3]
+except:
+    fmt = "Version $v, $u/$m Users, $p, $b"
```

Comment:
Why is this handled through an exception rather than a check-and-set with default value?

Reply:
I understand this to be more Pythonic. I can still change it though if you'd like.

Reply:
I changed it to be check-and-set.

Reply:
To add some context on (my) reasoning:

Conceptually exceptions are for exceptions as in exceptional situations and program flow. That is why I would not expect expected checks to be handled with exceptions.

There is a technical consequence to this too. I am not sure how python handles it, but in other languages exceptions come at a cost. An exception object is created. A simple check does not do that.